.gitignore

```diff
@@ -37,6 +37,8 @@
 /packages/flutter/coverage/
 version
 
+# Cookies
+.cookies
```

Comment:
Has the cookies/refresh token things been tested on a real phone? Where do we save the cookies?

Reply:
1. It has not been tested
2. It saves the refresh cookie on the device's file storage. I do not know where

Reply:
Moved to Hive storage 😃